src/Broadway/Bundle/BroadwayBundle/Command/SchemaEventStoreCreateCommand.php

```diff
@@ -28,15 +28,16 @@ class SchemaEventStoreCreateCommand extends DoctrineCommand
      */
     protected function configure()
     {
+        $name = 'broadway:event-store:schema:init';
         $this
-            ->setName('broadway:event-store:schema:init')
+            ->setName($name)
             ->setDescription('Creates the event store schema')
             ->setHelp(
 <<<EOT
-The <info>broadway:event-store:schema:init</info> command creates the schema in the default
+The <info>$name</info> command creates the schema in the default
```

Comment:
i think for symfony commands there is a placeholder special for this that must be used instead